### ISSUE TITLE
Sharing Articles 

### ISSUE THREAD

**Issue by Videhi** [shescoding/shescoding-dot-org#44]
1) Click an article here:  http://shescoding-wamuq6fved.elasticbeanstalk.com/resources
2) The corresponding article page opens up (Currently, it shows test data)
3) I couldn't find a means to share the article like a Twitter/LinkedIn button. Please do provide them to enhance traffic.

**Comment by natsteinmetz** [comment]
@Videhi Yes, this is a feature we have planned for the (hopefully) close future. Thanks! :)

**Comment by natsteinmetz** [comment]
This is in PR #85
